libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -114,10 +114,26 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
 
     // @Param: _W_RSSI
     // @DisplayName: RSSI warn level (in %)
-    // @Description: Set level at which RSSI item will flash
-    // @Range: 0 99
+    // @Description: Set level at which RSSI item will flash (in positive % or negative dBm values as applicable). 30% or -100dBm are defaults.
+    // @Range: -128 100
     // @User: Standard
-    AP_GROUPINFO("_W_RSSI", 12, AP_OSD, warn_rssi, 30),
+    AP_GROUPINFO("_W_RSSI", 12, AP_OSD, warn_rssi, AP_OSD_WARN_RSSI_DEFAULT),
+
+#if AP_OSD_CRSF_PANELS_ENABLED
```

Comment:
Please move these to the bottom - it makes it very hard for future editors to add new parameters if they don't know that some of these indexes are used

Reply:
Ok, done!!! 

Indeed, good catch to make life easier for future changes. :-)